pkg/controllererrors/errorcodes.go

```diff
@@ -30,7 +30,8 @@ const (
 
 	// k8sContext package
 	ErrorEnpdointsNotFound              ErrorCode = "ErrorEnpdointsNotFound"
-	ErrorFetchingEnpdoints              ErrorCode = "ErrorFetchingEnpdoints"
+	ErrorFetchingEnpdoints              ErrorCode = "ErrorFetchingEndpoints"
```

Comment:
can you correct ErrorFetchingEnpdoints to ErrorFetchingEndpoints. Seems like a spelling mistake :/